packages/server/lib/controllers/connection.controller.ts

```diff
@@ -695,9 +697,12 @@ class ConnectionController {
                         id: updatedConnection.id,
                         connection_id,
                         provider_config_key,
-                        environment_id: environmentId
+                        environment_id: environmentId,
+                        auth_mode: template.auth_mode,
+                        operation: updatedConnection?.operation ? updatedConnection.operation : AuthOperation.UNKNOWN
```

Comment:
`operation:  updatedConnection?.operation || AuthOperation.UNKNOWN`

Reply:
Yeah, saw that as well when I was doing a personal review. Will update. Thanks!

Reply:
Updated with https://github.com/NangoHQ/nango/pull/1585/commits/89719c7688f4c7458c30da1386abdb858b94c4c7